examples/huggingface/transcribe.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from pyctcdecode import build_ctcdecoder
+import torch
+import torchaudio
+from transformers import pipeline
+from transformers import AutoConfig, AutoModelForSpeechSeq2Seq
+from reverb_hf import ReverbModel
+from reverb_config import ReverbConfig
+from reverb_processor import ReverbFeatureExtractor, ReverbTokenizer
+
+
+AutoConfig.register("reverb_asr", ReverbConfig)
+AutoModelForSpeechSeq2Seq.register(ReverbConfig, ReverbModel)
+feature_extractor = ReverbFeatureExtractor(return_tensors='pt')
+tokenizer = ReverbTokenizer(
+    "hf-reverb/tk.model",
+)
+decoder_ids = []
+with open("hf-reverb/tk.units.txt", 'r') as units_file:
```

Comment:
same as above, can we include this modification in the preprocessing step so the inference is as simple as possible?